radixdlt-java/src/main/java/com/radixdlt/client/atommodel/message/MessageParticle.java

```diff
@@ -56,24 +56,40 @@ private MessageParticle() {
 	@DsonOutput(DsonOutput.Output.ALL)
 	private RadixAddress from;
 
+	/**
+	 * Metadata, aka data about the data (e.g. contentType).
+	 * Will consider down the line whether this is worth putting
+	 * into a more concrete class (e.g. MetaData.java).
+	 */
+	@JsonProperty("metaData")
+	@DsonOutput(DsonOutput.Output.ALL)
+	private Map<String, String> metaData = new TreeMap<>();
```

Comment:
`TreeMap` isn't required here, as the serializer sorts keys on output.

Is "final" an option here?